tests/src/mixer_test.cc

```diff
@@ -288,7 +288,7 @@ TEST_P(MixerControlsTest, Event) {
 
     EXPECT_EQ(mixer_wait_event(mixer_object, 1000), 1);
 
-    EXPECT_EQ(mixer_consume_event(mixer_object), 0);
+    EXPECT_EQ(mixer_consume_event(mixer_object), 1);
```

Comment:
For anyone else reading this, 1 means success, 0 means no pending event, and -errno may be returned in case of an error (according to the doc string).